### PR TITLE
openblas: add 0.3.13

### PR DESCRIPTION
**Pull request by michaelkuhn**

### CODE REVIEW

File: var/spack/repos/builtin/packages/openblas/package.py [hunk 1]
```diff
@@ -274,6 +275,9 @@ def make_defs(self):
         if self.spec.satisfies('%gcc@:4.8.4'):
             make_defs.append('NO_AVX2=1')
 
+        if self.spec.satisfies('@0.3.13:'):
+            make_defs.append('RANLIB=ranlib')
```
Comment (haampie) [inline var/spack/repos/builtin/packages/openblas/package.py:281]: Out of curiosity, why was this necessary? Maybe a comment with the explanation would be useful?

Comment (michaelkuhn) [reply var/spack/repos/builtin/packages/openblas/package.py:281]: Sorry, totally forgot to add some comment about this. Should be fixed now.

**Review by alalazo — APPROVED** [review]

File: var/spack/repos/builtin/packages/openblas/package.py (hunk 1 above)
Comment (alalazo) [reply var/spack/repos/builtin/packages/openblas/package.py:281]: Reading the comment. Was it advertized as a feature or a bug upstream? :slightly_smiling_face:
